src/main/java/hudson/plugins/jira/JiraBuildAction.java

```diff
@@ -11,11 +15,12 @@
  * @author Kohsuke Kawaguchi
  */
 public class JiraBuildAction implements Action {
-    public final AbstractBuild<?, ?> owner;
+    
+	public final Run<?, ?> owner;
```

Comment:
@oleg-nenashev as I understand you mean other Jenkins plugins that depend on this one, are you aware of any? (just by chance). So, we should keep the old API with @deprecated for one release along with introducing the new API and that would be sufficient, right?
Also, I'm curious how do you know about RunAction2, that's good to know, but I don't see any description in the [API doc for this](http://javadoc.jenkins-ci.org/jenkins/model/RunAction2.html). Is this just your Jenkins core source knowledge from exp? :)